sessions/recap-project-6-NEW/activity-planner/user-stories/2_activity-details.md

```diff
@@ -0,0 +1,35 @@
+# Activity Details
+
+## Value Proposition
+
+**As a user**
+
+**I want to** access detailed information about each activity,
+
+**so that** I can make informed decisions and plan my activities effectively.
+
+## Description
+
+Group Todo: Add wireframes
+
+## Acceptance Criteria
+
+- Each activity in the activities list has a link to a details page using MongoDB's unique document ID.
+- The details page fetches specific activity data from MongoDB using the activity ID.
+- Each details page contains:
+  - Activity title
+  - Activity image
+  - Activity description
+  - Activity categories
+  - Activity area
+  - Activity country
+  - Creation date
+  - Last update date
+- The detailed view includes a "back" link, allowing users to easily navigate back to the list of activities.
+- The page implements proper error handling for non-existent activity IDs.
+- The page shows a loading state while fetching activity details.
+
+## Tasks
+
+- [ ] Create feature branch `feature/activity-details`
+- [ ] Group Todo: Add tasks
```

Comment:
We decided in the team that we want the following "reduction" of the user stories: 
0. Full
1. Full
2. Remove wireframe
3. Remove wireframe, Tasks
4. Remove wireframe, Tasks, AC

Here in the 2. US wireframe as well as Tasks are missing. We need to add the tasks here again, as well as the AC in the 3. US.